pkg/cmd/server/admin/create_servercert.go

```diff
@@ -93,6 +102,13 @@ func (o CreateServerCertOptions) Validate(args []string) error {
 		return errors.New("key must be provided")
 	}
 
+	if len(o.ExpirationDate) > 0 {
+		if _, err := time.Parse(time.RFC3339, o.ExpirationDate); err != nil {
+			return fmt.Errorf("invalid expiration date: %v", err)
```

Comment:
Unfortunately, when we're using `time.Parse()` with `time.RFC3339` format, user have to provide date and time in the format like `2017-01-01T00:00:00Z`. And even `Z` postfix here is mandatory:
```console
error: invalid expiration date: parsing time "2017-01-01T00:00:00" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "Z07:00"
```
Probably, we should use custom date layout. Do we have requirement on that?